src/Components/EnergyCalculator/Steps/Utilities.tsx

```diff
@@ -0,0 +1,100 @@
+import { FormattedMessage } from 'react-intl';
+import QuestionHeader from '../../QuestionComponents/QuestionHeader';
+import QuestionQuestion from '../../QuestionComponents/QuestionQuestion';
+import QuestionDescription from '../../QuestionComponents/QuestionDescription';
+import RHFOptionCardGroup from '../../RHFComponents/RHFOptionCardGroup';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { useParams } from 'react-router-dom';
+import { ReactComponent as Plug } from '../Icons/Plug.svg';
+import { ReactComponent as LowFuel } from '../Icons/LowFuel.svg';
+import PrevAndContinueButtons from '../../PrevAndContinueButtons/PrevAndContinueButtons';
+import { useDefaultBackNavigationFunction, useGoToNextStep } from '../../QuestionComponents/questionHooks';
+import { useContext } from 'react';
+import { Context } from '../../Wrapper/Wrapper';
+import useScreenApi from '../../../Assets/updateScreen';
+
+const Utilities = () => {
+  const { formData, setFormData } = useContext(Context);
+  const { uuid } = useParams();
+  const { updateScreen } = useScreenApi();
+  const backNavigationFunction = useDefaultBackNavigationFunction('energyCalculatorUtilityStatus');
+  const nextStep = useGoToNextStep('energyCalculatorUtilityStatus');
+  const utilityStatusOptions = {
+    electricityIsDisconnected: {
+      icon: <Plug className="option-card-icon" />,
+      text: {
+        props: {
+          id: 'utilityStatusOptions.electricityIsDisconnected',
+          default_message: 'Your electricity and/or gas has been disconnected.',
+        },
+      },
+    },
+    hasPastDueEnergyBills: {
+      icon: <LowFuel className="option-card-icon" />,
+      text: {
+        props: {
+          id: 'utilityStatusOptions.hasPastDueEnergyBills',
+          default_message: 'You have a past-due electric or heating bill or you are low on fuel.',
+        },
+      },
+    },
+  };
+
+  const formSchema = z.object({
+    energyCalculator: z.object({
+      electricityIsDisconnected: z.boolean(),
+      hasPastDueEnergyBills: z.boolean(),
+    }),
+  });
+
+  const { handleSubmit, watch, setValue } = useForm<z.infer<typeof formSchema>>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      energyCalculator: {
+        electricityIsDisconnected: formData.energyCalculator?.electricityIsDisconnected ?? false,
+        hasPastDueEnergyBills: formData.energyCalculator?.hasPastDueEnergyBills ?? false,
+      },
+    },
+  });
+
+  const formSubmitHandler: SubmitHandler<z.infer<typeof formSchema>> = (memberData) => {
+    if (uuid === undefined) {
+      throw new Error('uuid is not defined');
+    }
```

Comment:
This should fix the TS errors.
```suggestion
    }
    if (!energyDataAvailable) {
      throw new Error('energy data is not set up');
    }
```